python/shipDigiReco.py

```diff
@@ -931,35 +931,37 @@ def findTracks(self):
 #check
     atrack = entry[1]
     theTrack = entry[0]
-    if not theTrack.checkConsistency():
-     print('Problem with track before fit, not consistent',atrack,theTrack)
-     continue
+    try:  theTrack.checkConsistency()
+    except:  print('Problem with track before fit, not consistent',atrack,theTrack)
 # do the fit
     try:  self.fitter.processTrack(theTrack) # processTrackWithRep(theTrack,rep,True)
     except:
-      if global_variables.debug:
-        print("genfit failed to fit track")
+      if global_variables.debug:  print("genfit failed to fit track")
       error = "genfit failed to fit track"
       ut.reportError(error)
       continue
 #check
-    if not theTrack.checkConsistency():
-     if global_variables.debug:
-       print('Problem with track after fit, not consistent', atrack, theTrack)
-     error = "Problem with track after fit, not consistent"
-     ut.reportError(error)
-     continue
+    try:  theTrack.checkConsistency()
+    except:
+      if global_variables.debug:  print('Problem with track after fit, not consistent', atrack, theTrack)
+      error = "Problem with track after fit, not consistent"
+      ut.reportError(error)
     try:
       fittedState = theTrack.getFittedState()
       fittedMom = fittedState.getMomMag()
     except:
       error = "Problem with fittedstate"
       ut.reportError(error)
       continue
-    fitStatus   = theTrack.getFitStatus()
-    nmeas = fitStatus.getNdf()
-    chi2        = fitStatus.getChi2()/nmeas
-    global_variables.h['chi2'].Fill(chi2)
+    try:
```

Comment:
The try/except block should only wrap the `isFitConverged` method, which is the one that can throw an exception.

While we're touching these lines, please 1 (and only 1) space around operators and 4 spaces indentation per level.